.claude/hooks/post_tool_use.py

```
@@ -0,0 +1,51 @@
+#!/usr/bin/env -S uv run --script
+# /// script
+# requires-python = ">=3.8"
+# ///
+
+import json
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)